sag_desc.par

```diff
@@ -59,19 +59,19 @@ Charon's boathook
 /|
 /
 5-comma, syntonic comma (81/80)
-Didymus' dibbler
+Didymus' dibbler (Osiris' flail)
 '/|
 '/
-3-comma, Pythagorean comma (531441/524288)
+1-comma, Pythagorean comma (531441/524288)

```

Comment:
I saw this as the 1C in George's JI Calculator. I think since the 3's are dropped from all other comma names, this is more consistent. 

However, if we want to support consistent naming of other 3-limit commas, we'll need something else, or else they would all become 1 commas (or kleismas, or schismas, etc.)

When I was going through and learning about key 3-limit intervals w/r/t boundaries between commatic interval classes, I saw a proliferation of naming conventions for such intervals. 
| 27 -17 > being the 17-comma
| -30 19 > being the 19-tone comma
| -49 31 > being the 31-3-comma

I suppose I like the last style the best.

That would make this Pythagorean comma here the 12-3-comma.

Reply:
"17-comma" and "31-3-comma" are terrible names for these commas since they suggest the involvement of the primes 17 and 31. "19-tone comma" at least avoids this confusion, but it's modern expression should be "19-edo comma", or better still, "19-edo apotome-plus-3-comma" (since it's 137 cents). And in that scheme, [-19 12⟩ which is traditionally called THE Pythagorean comma, would be the 12-edo 3-comma.

In the same way that we can simply read "7-comma" as "septimal comma" (the traditional name for 63:64), we can read 3-comma as "Pythagorean comma". This doesn't work if we insist on mathematical consistency and call it the "1-comma". There is a strong association between prime 3 and "Pythagorean". There is no such association with the number 1 and "Pythagorean". At least "3-comma" is not _wrong_ mathematically. It's only redundant. You could also call a 5-comma a 15-comma, but there's no reason to do so.

Also, the 12-edo 3-comma can simply be called the "3-comma" and therefore read as "the Pythagorean comma" as is traditional, because it is the simplest 3-comma in the size category "comma". And [-30 19⟩ can simply be called the apotome-plus-3-comma (or 3-apotome-plus-comma) because it's the simplest 3-comma in the size category "apotome-plus-comma".

And [65 -41⟩ can be called the complex 3-comma. This is still being thrashed out here:
See http://forum.sagittal.org/viewtopic.php?f=12&t=484&p=1717#oneorthree
and http://forum.sagittal.org/viewtopic.php?p=1724#p1724

Reply:
I'll change it back to the 3-comma. I agree with your reasoning. Thanks for unpacking your thought process -- it's helped me get my head around this subtlety to the Keenan/Secor/Sagittal comma naming conventions.

I've been following that thread, and just commented! It is not relevant for continued decision-making on this tiny little comment thread, though, so I'll go ahead and mark this conversation as resolved.

Reply:
Thanks. 1/1 = 1u is a great solution.